shared/chat/conversation/messages/wrapper/wrapper.tsx

```diff
@@ -1,6 +1,6 @@
 import * as C from '@/constants'
 import * as Kb from '@/common-adapters'
-import * as React from 'react'
+gimport * as React from 'react'
```

Comment:
this shouldn't be there but i made my own MR instead of fixing this up tnx. https://github.com/keybase/client/pull/26097